### ISSUE TITLE
Add flags from sttm-api on bootup

### ISSUE THREAD

**Issue by bogas04** [KhalisFoundation/sttm-web#507]
We can't control features based on flags as of now.

**Comment by bogas04** [comment]
@ManjotS can we use sttm-api for feature flags?

**Comment by ManjotS** [comment]
yes of course, can you write a ticket, I can try and program with my limited js abilities

**Comment by saintsoldierx** [comment]
@ManjotS can you do this or do we need to re-assign it?

**Comment by saintsoldierx** [comment]
What would also be amazing is to have what raag time it is.

**Comment by bogas04** [comment]
Sorry I can't take any more tasks. Not able to balance it with work and
life.

**Comment by saintsoldierx** [comment]
This is the concept we have 
![image](https://user-images.githubusercontent.com/27791330/67148631-7a088a00-f256-11e9-9798-37446c2b1855.png)

**Comment by bogas04** [comment]
Make sure it scales to mobile view as well.

**Comment by ManjotS** [comment]
@saintsoldierx what is the sttm-api ticket you are referencing? I can implement once I know.

**Comment by ManjotS** [comment]
@saintsoldierx bump

**Comment by saintsoldierx** [comment]
@ManjotS I'm not sure, i didn't mention an API. But need to figure out how we can make these features available.

**Comment by ManjotS** [comment]
should we just add a key/value store service for this?

**Comment by saintsoldierx** [comment]
Yes, i think that could work. Whatever you think's best.

**Comment by ManjotS** [comment]
pending sttm-api/10

**Comment by saintsoldierx** [comment]
Could I get a little info as to how this will work? it looks like api work is done... @ManjotS

**Comment by ManjotS** [comment]
/settings returns all settings
/settings/settingname returns the data for that option

The underlying table is the settings table in sttm database. It has setting_name and setting_value. You can persist any datatype there as a key/value. So it could be a json structure or a bool etc.

**Comment by saintsoldierx** [comment]
This is still relevant @gurjit03 - but i am not sure what exactly needs to be done here
